docs/sources/alerting/notifications.md

```diff
@@ -183,9 +183,9 @@ In DingTalk PC Client:
 
 5. In "Add Robot" panel, input a nickname for the robot and select a "message group" which the robot will join in. click "next".
 
-6. There will be a Webhook URL in the panel, looks like this: https://oapi.dingtalk.com/robot/send?access_token=xxxxxxxxx. Copy this URL to the grafana Dingtalk setting page and then click "finish".
+6. There will be a Webhook URL in the panel, looks like this: https://oapi.dingtalk.com/robot/send?access_token=xxxxxxxxx. Copy this URL to the grafana DingTalk setting page and then click "finish".
```

Comment:
Grafana not grafana?

Reply:
Hello @achatterjee-grafana , changes made as per suggestion